librecad/src/actions/rs_actioneditcopy.h

```diff
@@ -7,7 +7,7 @@
 **
 **
 ** This file may be distributed and/or modified under the terms of the
-** GNU General Public License version 2 as published by the Free Software 
+** GNU General Public License version 2 as published by the Free Software
```

Comment:
Note `virtual void mouseMoveEvent(QMouseEvent* e);`

Some git GUIs make it easy to revert the file 
https://git-scm.com/downloads/guis
or https://git-scm.com/book/en/v2/Git-Basics-Undoing-Things